### PR TITLE
Simplified library and fixed issue with wrong detektor.js lib name.

### PR DESCRIPTION
**Pull request by unckleg**
- Added .brackets.json
- Added .editorconfig
- Added .idea to .gitignore

- Replaced double quotes to single quotes
- Replaced tab chars to 2 spaces
- Simplified library logic
- Fixed issue with **odredi** (_detektor-js_) invalid npm registry path

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,193 +1,177 @@
-import odredi from '@pionir/detektor.js';
+import odredi from '@pionir/detektor-js';
```
Comment (unckleg) [inline main.js:1]: `import odredi from '@pionir/detektor.js';` 

Doesn't exist on npm registry while '@pionir/detektor-js' does and is valid one.

File: main.js [hunk 2]
```diff
@@ -1,193 +1,177 @@
...
+
+// a map containing all of the characters and exceptions
+const convMap = new Map([
```
Comment (unckleg) [inline main.js:4]: Moved static map from **preslovi** method definition to module definition.

File: main.js [hunk 3]
```diff
@@ -1,193 +1,177 @@
...
-            for (let except of exceptions) {
-                convMap.set(except, changer(except));
-            }
-        }
-        else {
-            for (let except of exceptions) {  //adds them the opposte way otherwise
-                convMap.set(changer(except), except);
-            }
-        }
+  const tester = !type ? function (word) {
```
Comment (unckleg) [inline main.js:145]: Simplified **preslovi** method logic.

File: tests/test.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import preslovi from '@pionir/preslovljivac-js';
+import preslovi from '../main';
```
Comment (unckleg) [inline tests/test.js:1]: Tests in the published packages should point to relative paths/files instead of npm library resolving.

**Comment by Njolf** [comment]
Hey, I appreciate that you took time to review my library and give feedback. Also, thanks for pointing out the flaw in importing *detektor-js* and the test file imports, they're really helpful (not sure how the *detektor-js* one flew under the radar in my tests, honestly). However, I don't see the need for ".editorconfig" and such yet - on an individual level, they're useful, but I'm not sure about having them in the repository. Other than that, I appreciate the time that you took to make the code simpler and prettier, but I'd prefer there to be no ternary operators (if that's okay). Anyway, since the functionality issues are a big problem, I'll make my own commit with just those fixes right now, so we have time for the rest later.

**Pull request closed without merging**